validator/validator-main.protoascii

```diff
@@ -1494,7 +1494,29 @@ tags: {
     value_regex: "[0-9]*"
   }
 }
-# 4.4.8 The dl element
+# 4.11.1 (HTML5.3) The details element
+tags: {
+  html_format: AMP
+  html_format: AMP4ADS
```

Comment:
Lets begin with just `AMP` and leave the other formats for future PRs. If you believe these should be added to other formats then I'd recommend opening an issue asking for feedback from others as each one of these formats have a different spec. Same for below for `SUMMARY`.

Reply:
`EXPERIMENTAL` would also be removed?

Reply:
Yes